src/test/java/gov/nih/nci/evs/api/service/TermSuggestionFormServiceTest.java

```diff
@@ -59,49 +61,56 @@ public class TermSuggestionFormServiceTest {
   private final String subject = "Test Subject";
   private final String msgBody = "Test Body";
 
+  // Config url
+  private final String configUrl =
```

Comment:
We want to avoid raw URLs in the code where possible. This can use the config value in the yaml file, accessible from [ApplicationProperties.java](https://github.com/NCIEVS/evsrestapi/blob/34ee1d330a4e54a1c4d21a4cd5fa6a14a692c0a4/src/main/java/gov/nih/nci/evs/api/properties/ApplicationProperties.java#L107) You can find a sample usage of this in getWelcomeText in [BaseLoaderService](https://github.com/NCIEVS/evsrestapi/blob/34ee1d330a4e54a1c4d21a4cd5fa6a14a692c0a4/src/main/java/gov/nih/nci/evs/api/service/BaseLoaderService.java#L483).

Reply:
Updated to pull from application-test.yml